### PR TITLE
Translate "Ruby 3.3.0-rc1 Released" (ko) 

### PR DESCRIPTION
**Pull request by riseshia**
:link: #2818

Actual diff is  8f3b70dd0

### CODE REVIEW

File: ko/news/_posts/2023-12-11-ruby-3-3-0-rc1-released.md [hunk 1]
```diff
@@ -0,0 +1,315 @@
...
+layout: news_post
+title: "Ruby 3.3.0-rc1 릴리스"
+author: "naruse"
+translator: "shia"
+date: 2023-12-11 00:00:00 +0000
+lang: ko
+---
+
+{% assign release = site.data.releases | where: "version", "3.3.0-rc1" | first %}
+Ruby {{ release.version }} 릴리스를 알리게 되어 기쁩니다. Ruby 3.3은 Prism이라고 명명된 새 파서를 추가하고, 파서 생성기로 Lrama를 사용하며, RJIT으로 명명된 새로운 순수 Ruby JIT 컴파일러를 추가하고, 특히 YJIT에서 많은 성능 향상이 있습니다.
```
Comment (marocchino) [inline ko/news/_posts/2023-12-11-ruby-3-3-0-rc1-released.md:11]: maybe)
I believe less Hanja improve readability. 
Prism이라고 명명된 -> Prism이라는 이름의

File: ko/news/_posts/2023-12-11-ruby-3-3-0-rc1-released.md [hunk 2]
```diff
@@ -0,0 +1,315 @@
...
+
+RC1 릴리스 이후로 ABI의 호환성을 깨트리는 변경은 가급적 피할 것입니다. 그래야 한다면, 릴리스 노트에서 해당 내용을 발표할 예정입니다.
```
Comment (marocchino) [inline ko/news/_posts/2023-12-11-ruby-3-3-0-rc1-released.md:13]: maybe）context的に
```suggestion
RC1 릴리스 이후로 ABI의 하위호환성을 깨트리는 변경은 가급적 피할 것입니다. 그래야 한다면, 릴리스 노트에서 해당 내용을 발표할 예정입니다.
```

**Review by marocchino — APPROVED** [review]
👍🏽

File: ko/news/_posts/2023-12-11-ruby-3-3-0-rc1-released.md (hunk 1 above)
Comment (riseshia) [reply ko/news/_posts/2023-12-11-ruby-3-3-0-rc1-released.md:11]:
```suggestion
Ruby {{ release.version }} 릴리스를 알리게 되어 기쁩니다. Ruby 3.3은 Prism이라는 이름의 새 파서를 추가하고, 파서 생성기로 Lrama를 사용하며, RJIT으로 명명된 새로운 순수 Ruby JIT 컴파일러를 추가하고, 특히 YJIT에서 많은 성능 향상이 있습니다.
```

File: ko/news/_posts/2023-12-11-ruby-3-3-0-rc1-released.md (hunk 2 above)
Comment (riseshia) [reply ko/news/_posts/2023-12-11-ruby-3-3-0-rc1-released.md]: +1 fixed 68d9a20

File: ko/news/_posts/2023-12-11-ruby-3-3-0-rc1-released.md (hunk 1 above)
Comment (riseshia) [reply ko/news/_posts/2023-12-11-ruby-3-3-0-rc1-released.md]: +1 fixed 68d9a20

**Comment by riseshia** [comment]
Thanks for review!

**Pull request merged**